man/fi_endpoint.3.md

```diff
@@ -400,13 +400,15 @@ parameter, only one will be canceled.  In this case, the operation
 which is canceled is provider specific.  The cancel operation is
 asynchronous, but will complete within a bounded period of time.
 
-## fi_alias
+## fi_ep_alias
 
 This call creates an alias to the specified endpoint.  Conceptually,
 an endpoint alias provides an alternate software path from the
 application to the underlying provider hardware.  Applications
 configure an alias endpoint with data transfer flags, specified
-through the fi_alias call.  Typically, the data transfer flags will be
+through the fi_ep_alias call. The flags must include FI_SEND or FI_RECV
+(not both) with other flags OR'ed to indicate the type of data transfer the flags should
+apply to. Typically, the data transfer flags will be
```

Comment:
Let's call out FI_TRANSMIT in place of FI_SEND (they're interchangeable), and document that these override the op_flags set during EP creation as part of the transmit and receive attributes.  That will help clarify how the alias EP differs from the base EP.